delta/app/src/main/resources/app.conf

```diff
@@ -242,6 +241,7 @@ app {
       ask-timeout = 25 seconds
       evaluation-max-duration = 14 seconds
       stash-size = 10
+      retry = ${app.defaults.never-retry-strategy}
```

Comment:
Could we have the default as `constant-retry-strategy` ?

Reply:
But then we'll have double retries: once in `ShardedAggregate` and once in `IndexingSource`?

Reply:
Yes. I don't think that's a problem since IndexingSource should retry for a long time anyway.
And if we cannot do that, what's the point of having the possibility to retry here at all?